test/typedarray21.js

```diff
@@ -0,0 +1,22 @@
+// generator: none
```

Comment:
oh, no you aren't.  there are a few other tests that fail for similar
reasons.

On Mon, Aug 10, 2015 at 3:08 AM, Carlos Alberto Cortez <
notifications@github.com> wrote:

> In test/typedarray21.js
> <https://github.com/toshok/echojs/pull/87#discussion_r36617072>:
>
> > @@ -0,0 +1,22 @@
> > +// generator: none
>
> It does - but the resulting code doesn't run in Node (yet):
>
> TypeError: Object # has no method 'copyWithin'
>
> Or am I missing something? Let me know ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/toshok/echojs/pull/87/files#r36617072>.
>